src/Manifest/Manifest.router.ts

```diff
@@ -95,8 +95,11 @@ export class ManifestRouter extends Router {
   getProjectManifest = (req: Request, res: Response) => {
     const id = server.extractFromReq(req, 'id')
     const project = new S3Project(id)
+
+    res.setHeader('Cache-Control', 'public,max-age=31536000,immutable')
```

Comment:
What do you think of moving `31536000`  to something like [`DEFAULT_ASSET_PACK_CACHE`](https://github.com/decentraland/builder-server/blob/master/src/AssetPack/AssetPack.router.ts#L28)

Reply:
that will work too.

I was thinking also that we can have a new middleware like `withInmutableCache` to be used directly on each router